src/main/java/com/vmware/vipclient/i18n/base/HttpRequester.java

```diff
@@ -60,7 +61,17 @@ public void setBaseURL(String baseURL) {
     private Map<String, String> customizedHeaderParams = null;
```

Comment:
Change this to static and add another instance variable to store cache control header params?

Reply:
Actually, header request parameters may vary per request. This should be a local variable in the request method and must be set per request. I did not want to change this for now because I wasn't sure why this has been an instance variable. Are we using this setCustomizedHeaderParams anywhere before?